incubator/bitcoind/templates/deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: extensions/v1beta1
```

Comment:
`apps/v1beta2`

We officially support 1.8+.

Reply:
Fixed